README.md

```diff
@@ -57,6 +57,20 @@ The tooling has evolved over time and therefore when generating new packages, yo
 This is because the new package has a dependency on an existing package, it was regenerated and changes were detected from
 when it was originally generated.
 
+#### Workflow
```

Comment:
Consider utilizing a linter to validate the correctness of the MD.  There are several corrections that should be made.

```suggestion
#### Workflow

```